.circleci/config.yml

```diff
@@ -384,6 +384,12 @@ jobs:
       - DEPLOY_ENV: uat
     steps: *deploy_steps
 
+  deploy_aws_dev:
```

Comment:
Why do we keep this? I see it runs OK in the test builds already.

Reply:
@dincho not sure what do you mena, but you have there deploy_intenrgariton deploy_Dev1 etc... so i think we need one for deplou_aws_dev for specyfiint dev_aws env?

Reply:
no, once we're ready we'll just tag AWS stack with `tag_env_uat` for example